client/components/GameList.jsx

```diff
@@ -5,7 +5,12 @@ const GameList = (props) => (
   <div className="game-list" >
     <div className="game-list-heading">List of Related Games!</div>
     {props.games.map((game, index) => (
-      <GameListEntry game={game} key={index}/>
+      <GameListEntry 
+      game={game} 
```

Comment:
Lines 9 through 12 should be indented one more level so that it's easy to tell at a glance that the opening `<GameListEntry` and closing `/>` encapsulate the properties being defined within.